### DISCUSSION TITLE
Can we have configurable colors?

### DISCUSSION THREAD

**Discussion by AucT** [twbs/bootstrap#34986] [Ideas]
Css support variables few years.
Day night themes are supported in all major browsers few years.
Why in 2k21 bootstrap don't use configurable primary color and other colors it uses. This will make coding night theme a breath.

Can you explain, maybe I don't know something.

**Comment by jonasbirkelof** [comment]
It's pretty simple to change with the scss files. Just go to the _variables.scss file and scroll a couple of rows to change the colors. Then compile and run an autoprefixer. I have made a version where I have created three custom color variables/theme colors, merged them with the theme-colors map and then I can have stuff like: `<button class="btn btn-pink">Button</button>` to have a pink button (and everything else that uses contextual colors).
